packages/handler-kit-express/src/logger.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-console */
+import * as L from "@pagopa/logger";
+import type {} from "@pagopa/logger-express";
+
+// Derive a concrete implementation L.Logger using Express.Request.log, if present, otherwise using console
```

Comment:
since express as no `native` logging, I wouldn't derive a `L.Logger` based on `console.log`

imho it would be better to fails with an error if req.log  it's not present